pillar/edx/init.sls

```diff
@@ -31,5 +31,5 @@ schedule:
     function: state.sls
     args:
       - edx.run_ansible
-      kwargs:
-        pillar: "{'edx': {'ansible_flags':  '--tags install:configuration'}}"
+    kwargs:
+      pillar: "{'edx': {'ansible_flags':  '--tags install:configuration'}}"
```

Comment:
The actual pillar data should be structured as a YAML dict under the pillar key.

Reply:
This still needs to be modified to be a yaml structure rather than a string.
```suggestion
      pillar:
        edx:
          ansible_flags: '--tags install:configuration'
```